common/lib/xmodule/xmodule/course_module.py

```diff
@@ -712,6 +712,12 @@ class CourseFields(object):
         scope=Scope.settings,
         default=""
     )
+    web_view_certificates = Boolean(
```

Comment:
let's rename this setting to 'cert_html_view_enabled', to match with the parameter below